### PR TITLE
Split school reports into tabs

### PR DESCRIPTION
**Pull request by dting**

### CODE REVIEW

File: server/api/student/student.controller.js [hunk 1]
```diff
@@ -147,33 +143,37 @@ exports.outreachSummary = function(req, res) {
...
     if (err) handleError(res, err);
-    Student.populate(results, {path: 'student'})
-      .then(function(withStudent) {
-        School.populate(withStudent, 
-          {path: 'school'})
-            .then(function(final) {
-              return res.status(200).json(final);
-            });
-      });
+    Intervention.populate(results, {
```
Comment (pdotsani) [inline server/api/student/student.controller.js]: Why the change in the model query from Student to Intervention?

**Comment by pdotsani** [comment]
lgtm.

File: server/api/student/student.controller.js (hunk 1 above)
Comment (dting) [inline server/api/student/student.controller.js]: The aggregation is on Intervention. Populate should use Intervention as well even though you are populating a school. In the case you aren't only populating a student from the intervention aggregation, e.g. a student and a school, using Student to populate doesn't really make sense. It also makes it so you don't need to require the Student model in this file if it isn't used.
